### ISSUE TITLE
Would it be possible to make a release to address issue with "wildcard tag protection" ? 

### ISSUE THREAD

**Issue by primeroz** [gitlabhq/terraform-provider-gitlab#175]
### Terraform Version
`Terraform v0.12.5`

### Affected Resource(s)
- gitlab_tag_protection

### Terraform Configuration Files
```hcl
resource "gitlab_tag_protection" "tag" {
  project             = "1234"
  tag                 = "v*"
  create_access_level = "maintainer"
}
```

### Debug Output
```
2019-09-16T17:05:22.768+0200 [DEBUG] plugin.terraform-provider-gitlab_v2.2.0_x4: 2019/09/16 17:05:22 [DEBUG] read gitlab tag protection for project 7058517,
 tag v*                                                                                                                                                     
2019-09-16T17:05:22.769+0200 [DEBUG] plugin.terraform-provider-gitlab_v2.2.0_x4: 2019/09/16 17:05:22 [DEBUG] GitLab API Request Details:                    
2019-09-16T17:05:22.769+0200 [DEBUG] plugin.terraform-provider-gitlab_v2.2.0_x4: ---[ REQUEST ]---------------------------------------                      
2019-09-16T17:05:22.769+0200 [DEBUG] plugin.terraform-provider-gitlab_v2.2.0_x4: GET /api/v4/projects/7058517/protected_tags/v%252A HTTP/1.1                
2019-09-16T17:05:22.769+0200 [DEBUG] plugin.terraform-provider-gitlab_v2.2.0_x4: Host: gitlab.com                                                           
2019-09-16T17:05:22.770+0200 [DEBUG] plugin.terraform-provider-gitlab_v2.2.0_x4: User-Agent: go-gitlab                                                      
2019-09-16T17:05:22.770+0200 [DEBUG] plugin.terraform-provider-gitlab_v2.2.0_x4: Accept: application/json                                                   
2019-09-16T17:05:22.770+0200 [DEBUG] plugin.terraform-provider-gitlab_v2.2.0_x4: Private-Token: XXXXXXXXXXXXXXXXXXXXx                                       
2019/09/16 17:05:22 [TRACE] <root>: eval: *terraform.EvalRefresh                                                                                            
2019-09-16T17:05:22.770+0200 [DEBUG] plugin.terraform-provider-gitlab_v2.2.0_x4: Accept-Encoding: gzip                                                      
2019-09-16T17:05:22.770+0200 [DEBUG] plugin.terraform-provider-gitlab_v2.2.0_x4:                                                                            
2019-09-16T17:05:22.770+0200 [DEBUG] plugin.terraform-provider-gitlab_v2.2.0_x4:                                                                            
2019-09-16T17:05:22.770+0200 [DEBUG] plugin.terraform-provider-gitlab_v2.2.0_x4: ----------------------------------------------------- 
```

### Issue description
`v2.2.0` was released with version `github.com/xanzy/go-gitlab v0.18.0` , this caused a double escape of protected tag which is breaking any use of wildcards in tag protection.

this was fixed in https://github.com/terraform-providers/terraform-provider-gitlab/commit/39047ee952a0a8d053eef767eebcc404e50f8bf0 but has not been released yet.

### Expected Behavior
Working `tag protection`

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

### Conclusions 
this is quite a big issue, i think that especially for tag protection wildcards are widely used ? I sure do 

I am in the process of getting our gitlab bunch of repos configured with terraform but got stuck at this.

I am currently running a local build out of master which is working fine but it won't scale for the rest of my team of course.

When can we expect a release to include this Fix ? 
Is it reasonable to ask for a patch release with this fix in ? 

Thanks in advance

**Comment by rhysjtevans** [comment]
Hit this issue as well, any eta for a release?

**Comment by roidelapluie** [comment]
Just the time for me to make the changelog and I will make a release this week

**Comment by wesselvdv** [comment]
> Just the time for me to make the changelog and I will make a release this week

You cutting a release as we speak? I see you just updated the changelog, I really need the envionment_scope option for a gitlab_project_variable.

**Comment by roidelapluie** [comment]
Yess I will ask for a release now

**Comment by armsnyder** [comment]
This was released in 2.3.0. Can we close this issue?

**Comment by armsnyder** [comment]
Closing as addressed in 2.3.0.
